Automattic/jetpack
# My Jetpack: Return error status for missing user connection

Properly inform an erro status if

* Product is active
* Product requires a user connection
* user connection is missing

#### Changes proposed in this Pull Request:

* My Jetpack: Return an error status for missing user connection

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install Backup, connect and activate the product
* Confirm the product shows as active
* Use the broken token tool and "clear user tokens"
* Go back to My Jetpack
* Confirm Backup card shows "Error"
